README.md

```diff
@@ -1,9 +1,9 @@
-# pints-models
+# bayesian-differential-equations-db
 
-The purpose of pints-models is to present Python implementations of scientific time series models from various disciplines.
+The purpose of bayesian-differential-equations-db is to present Python implementations of scientific time series models from various disciplines.
 
 While the `pints.toy` module already includes a number of simple models and distributions, this repository is intended for more complex models associated with real experimental data.
 
 ## Models and data
 
-Hydrology
+[Rainfall runoff model and river discharge data for the French Broad River at Asheville, North Carolina.](streamflow/)
```

Comment:
Can we include citations for the model? Also, can we create a sort of table that says why each of the models is interesting? So, here, I guess it's because of potential model misspecification / a potentially complex noise process / numerical error in solving the system.

Reply:
I like this idea, I started making a table which might work